cli/crates/cli/src/cli_input.rs

```diff
@@ -33,10 +33,10 @@ pub fn build_cli() -> Command<'static> {
             Command::new("init")
                 .args(&[
                     arg!([name] "the name of the project to create"),
-                    arg!(-t --template "the name of the template to use for the new project"),
+                    // arg!(-t --template "the name of the template to use for the new project"),
```

Comment:
Not a fan of commenting out code. That's what the git history is for. 😄

Reply:
Normally I'd be the one saying this, the idea here specifically is to outline the future interface (as the commented code below is doing). It's less of "I'm temporarily disabling this since it doesn't work" and more "this is the planned interface"

Reply:
We could still remove it, but I saw it as more of a description of the future additions